service/controller/v3/network/network.go

```diff
@@ -0,0 +1,65 @@
+package network
+
+import (
+	"context"
+	"net"
+
+	"github.com/giantswarm/ipam"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/azure-operator/service/controller/setting"
+	"github.com/giantswarm/azure-operator/service/controller/v3/key"
+)
+
+const (
+	ipv4Mask = 32
+)
+
+// Compute network subnets within network from CR.
```

Comment:
Proper godoc by convention starts with the thing it explains.

Reply:
done